packages/editor/src/components/post-type-support-check/test/index.js

```diff
@@ -10,27 +10,26 @@ import { PostTypeSupportCheck } from '../';
 
 describe( 'PostTypeSupportCheck', () => {
 	it( 'renders its children when post type is not known', () => {
-		let postType;
-		const tree = create(
-			<PostTypeSupportCheck postType={ postType } supportKeys="title">
+		const { container } = render(
+			<PostTypeSupportCheck postType={ undefined } supportKeys="title">
 				Supported
 			</PostTypeSupportCheck>
 		);
 
-		expect( tree.toJSON() ).toBe( 'Supported' );
+		expect( container ).toHaveTextContent( 'Supported' );
 	} );
 
 	it( 'does not render its children when post type is known and not supports', () => {
 		const postType = {
 			supports: {},
 		};
-		const tree = create(
+		const { container } = render(
 			<PostTypeSupportCheck postType={ postType } supportKeys="title">
 				Supported
 			</PostTypeSupportCheck>
 		);
 
-		expect( tree.toJSON() ).toBe( null );
+		expect( container ).not.toHaveTextContent( 'Supported' );
```

Comment:
Just like above, but inverted.

```suggestion
		expect( screen.queryByText( 'Supported' ) ).not.toBeInTheDocument();
```